app/services/publish/accredited_provider_updater.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Publish
+  class AccreditedProviderUpdater
+    attr_reader :provider, :new_accredited_provider, :recruitment_cycle
+
+    def initialize(provider_code:, recruitment_cycle_year:, new_accredited_provider_code:)
+      @recruitment_cycle = RecruitmentCycle.find_by!(year: recruitment_cycle_year)
+      @provider = Provider.find_by!(provider_code:, recruitment_cycle: @recruitment_cycle)
+      @new_accredited_provider = Provider.find_by!(provider_code: new_accredited_provider_code, recruitment_cycle: @recruitment_cycle)
+    end
+
+    def update_provider_and_courses
+      update_provider && update_courses
+    end
+
+    def update_provider
+      accredited_provider_enrichment = AccreditingProviderEnrichment.new(
+        {
+          UcasProviderCode: new_accredited_provider.provider_code,
+          Description: ''
```

Comment:
It is not the first time I see this description field on enrichments. Why do they need to be empty here?

Reply:
I really don't know but they are set with a description key and I wanted to make sure it wasn't going to break anything else 🤞

Reply:
Yes makes sense. Would be interesting to take a look at the database just to understand why. 🤔 🤔 🤔